### ISSUE TITLE
Can't compile Network/Google/Storage/Sync.hs

### ISSUE THREAD

**Issue by Bladtman242** [rrnewton/hgdata#6]
```
src/Network/Google/Storage/Sync.hs:326:103:
    Couldn't match expected type ‘time-1.5.0.1:Data.Time.Format.Locale.TimeLocale’
                with actual type ‘System.Locale.TimeLocale’
    NB: ‘time-1.5.0.1:Data.Time.Format.Locale.TimeLocale’
          is defined in ‘Data.Time.Format.Locale’ in package ‘time-1.5.0.1’
        ‘System.Locale.TimeLocale’
          is defined in ‘System.Locale’ in package ‘old-locale-1.0.0.7’
    In the first argument of ‘parseTime’, namely ‘defaultTimeLocale’
    In the second argument of ‘($)’, namely
      ‘parseTime defaultTimeLocale "%FT%T%QZ" lastModified’
cabal: Error: some packages failed to install:
handa-gdata-0.7.0.2 failed during the building phase. The exception was:
ExitFailure 1
```

It seems the locale package has been shuffled around since this was written.

**Comment by Bladtman242** [comment]
Cabal insists on getting 0.7.0.2 though, and this seems to be fixed in 0.7.0.3.

**Comment by rrnewton** [comment]
Thanks for the report -- yep I ran into that too and believe I've fixed it now.
